lib/build.js

```diff
@@ -344,19 +316,15 @@ function build(params) {
           Runtime: 'nodejs6.10',
           Timeout: 30
         }
-      },
-      SpotFleet: {
-        Name: params.LogicalName,
-        Type: 'Custom::SpotFleetFunction',
-        Properties: {
-          ServiceToken: cf.getAtt('SpotFleetFunction', 'Arn'),
-          SpotFleetRequestConfigData: params.Properties.SpotFleetRequestConfigData,
-          Region: params.Properties.SpotFleetRegion
-        }
       }
     }
   };
 
+  customFunctions[params.CustomFunctionName][params.LogicalName] = {
+    Type: cf.join(['Custom::', params.CustomFunctionName, 'Function']),
```

Comment:
So actually the `Type` only needs to be `Custom::Something`. This is enough for CloudFormation to know "this is not a resource type that I defined, its a custom one".

Once CloudFormation realizes that, then it looks for the `ServiceToken` -- this is the only property that CloudFormation requires exist in any custom resource. What you provide for this property is the pointer to a lambda function. This is how you define which resource is tied to which operation.